modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb

```diff
@@ -11,10 +11,10 @@
                      size: 40,
                      container_class: '-wide' %>
     <span class="form--field-instructions">
-      <%= t("storages.instructions.#{@storage.provider_type}.oauth_client_id") %>
-      <%= link_to "#{t("storages.provider_types.#{@storage.provider_type}")} / #{t("storages.instructions.#{@storage.provider_type}.administration")} / #{t("storages.instructions.#{@storage.provider_type}.oauth2_clients")}",
+      <%= "#{t("storages.instructions.copy_from")}: " %>
+      <%= link_to "#{t("storages.instructions.#{@storage.provider_type}.integration")}",
```

Comment:
@HDinger "navigation"? you mean "integration", right? However, you have a point. It is very important to mention "Nextcloud" as the user does not understand otherwise to go to Nextcloud and then into the settings there. Maybe sth. like "Copy this value from: Nextcloud Administration --> OpenProject"?

Reply:
Yes, good catch! That was a mistake.

Reply:
Fixed.